pkg/insight/filestore.go

```diff
@@ -68,6 +71,30 @@ func (s *Store) PutChunk(ctx context.Context, chunk Chunk) error {
 	return s.filestore.PutObject(ctx, path, data)
 }
 
+func (s *Store) LoadChunksCache(cache cache.Cache, projectID, appID string, kind model.InsightMetricsKind, step model.InsightStep, from time.Time, count int) ([]Chunk, error) {
```

Comment:
This should not be apart of `Store`.
Just an independent function as `LoadChunksFromCache`.